application/etc/database/changelog.yml

```
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: darshilshah2017
+      comment: "Create the 'partman' schema and pg_partman extension if they do not exist"
+      changes:
+        - tagDatabase:
+              tag: SAS-3A
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 14

(indentation)